core/src/main/java/com/google/cloud/sql/core/InstanceCheckingTrustManagerFactory.java

```diff
@@ -39,32 +40,38 @@
  * <p>class ConscryptWorkaroundTrustManager - the workaround for the Conscrypt bug.
  *
  * <p>class InstanceCheckingTrustManager - delegates TLS checks to the default provider and then
- * checks that the Subject CN field contains the Cloud SQL instance ID.
+ * does custom hostname checking in accordance with these rules:
+ *
+ * <p>If an instanceMetadata.casEnabled Check that the subjectAlternativeNames in the server
```

Comment:
nit: this sentence seems off to me, reads a bit funny...

Reply:
Good catch. I fixed it.